lib/timeutils/wallclocktime.c

```diff
@@ -22,12 +22,144 @@
  *
  */
 
+#include <ctype.h>
+#include <stdint.h>
+#include <string.h>
+
 #include "timeutils/wallclocktime.h"
 #include "timeutils/unixtime.h"
-#include "timeutils/strptime-tz.h"
 #include "timeutils/cache.h"
 #include "timeutils/misc.h"
 
+#define __UNCONST(a)    ((void *)(unsigned long)(const void *)(a))
+
+#define TM_YEAR_BASE 1900
+#define TM_SUNDAY     0
+#define TM_MONDAY     1
```

Comment:
Please retain the copyright/license of the original code

Reply:
Also, could you move this entire chunk towards the end of the file? I feel the main functions of the wall clock time struct are more important when approaching the file, and they get somewhat deemphasized with the strptime implementation or in front.

Reply:
Ooops, I forgot to move the license!

I am not sure to understand exactly what you meant in the second comment, so implemented what I understood.  Please tell me if I missed your point :wink: